pkg/glusterutils/peers_gd1.go

```diff
@@ -0,0 +1,79 @@
+package glusterutils
+
+import (
+	"encoding/xml"
+	"os/exec"
+)
+
+type peerGlusterd1 struct {
+	PeerID    string   `xml:"uuid"`
+	Hostname  []string `xml:"hostname"`
+	Connected int      `xml:"connected"`
+	State     int      `xml:"state"`
+	StateStr  string   `xml:"stateStr"`
+}
+
+type peersGlusterd1 struct {
```

Comment:
I feel it has some overlap with PR https://github.com/gluster/gluster-prometheus/pull/47 from Venkat.